portfolio/src/main/webapp/charts-script.js

```diff
@@ -0,0 +1,71 @@
+google.charts.load('current', {'packages':['corechart']});
+google.charts.setOnLoadCallback(drawCharts);
+
+/* drawCharts draws all of the charts to display. It is called in setOnLoadCallback. 
+The reasoning behind drawCharts is that setOnLoadCallback executes one function on load,
+ and drawCharts provides a centralized place for which to load all of the charts on the website.
+*/
+function drawCharts(){
+    //Since we draw charts from different pages, we only want to add in particular charts if they are defined on that page.
+    var pieChart = document.getElementById('pie-chart-container');
+    if (typeof(pieChart) != 'undefined' && pieChart != null){
+        drawPieChart();
+    }
+
+    var gwChart = document.getElementById('globalwarming-chart-container');
+    if (typeof(gwChart) != 'undefined' && gwChart != null) {
+        drawGlobalWarmingChart();
+    }
+
+}
+
+/** Creates a chart and adds it to the page. */
+function drawPieChart() {
+  const data = new google.visualization.DataTable();
+  data.addColumn('string', 'Pet Species');
+  data.addColumn('number', 'Count');
+        data.addRows([
+          ['Cats', 2],
+          ['Dogs', 2],
+          ['Reptiles', 4],
+          ['Birds', 3]
+        ]);
+
+  const options = {
```

Comment:
I wasn't sure what options meant until i looked at ur website

a better naming: chartDescription or something like that